.github/workflows/make-release.yml

```diff
@@ -0,0 +1,146 @@
+name: 'Kucher application builder'
+
+# Builds Kucher release
+
+on:
+  push:
+    branches:
+      - master
+
+# Ensures that only one workflow is running at a time
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  #
+  # Kucher test
+  #
+  kucher-test:
+    name: Test Kucher
+    if: startsWith(github.ref, 'refs/heads/')
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-2019]
+        python: ['3.10']
+    runs-on: ${{ matrix.os }}
+    steps:
+      # Checkout code from GitHub
+      - name: Check out
```

Comment:
Not to be nitpicking, but I always wondered what is the point of writing comments that restate the exact same thing as the code that follows.

Reply:
This is a style issue for maintenance. I prefer to enclose every step with a comment and a space so that they are distinguishable. The `name` tag is optional so it does not necessarily mean a similar comment will follow. But my comment was a bit lazy. I have amended to specify which repository is being checked out.